src/Data/Profunctor/Mapping.hs

```diff
@@ -34,16 +37,36 @@ class (Traversing p, Closed p) => Mapping p where
   -- 'dimap' 'Data.Functor.Identity.Identity' 'Data.Functor.Identity.runIdentity' '.' 'map'' ≡ 'id'
   -- @
   map' :: Functor f => p a b -> p (f a) (f b)
+  map' = roam collect
+
+  roam :: (forall f. (Distributive f, Applicative f)
```

Comment:
It would be much, much nicer to use `Representable` here than `Distributive` and `Applicative`, but that would incur an `adjunctions` dependency.